easybuild/framework/easyconfig/tools.py

```diff
@@ -111,13 +111,23 @@ def find_resolved_modules(unprocessed, avail_modules, retain_all_deps=False):
         new_ec = ec.copy()
         deps = []
         for dep in new_ec['dependencies']:
-            full_mod_name = ActiveMNS().det_full_module_name(dep)
+            full_mod_name = dep.get('full_mod_name', None)
```

Comment:
the naming scheme of external modules shouldn't matter, EB just needs to know about the module name so it can i) load the module, ii) include a `module load` statement in the generated module file

external modules are specified by *module name* only, no separate name/version (or suffix/toolchain) details, just a module name